tutorials/clustering.md

```diff
@@ -22,15 +25,92 @@ need a common superuser role on each postgres instance. This tutorial
 will assume the `postgres` role for this, but you may substitute any
 role common to all machines with superuser permission.
 
-Beyond this, please ensure that you've completed the following steps.
-Some or most of these may not be necessary depending on how you
-initially set up your instances:
-- Make sure you've enabled `max_prepared_transactions` on all the instances, this parameter is located in `postgresql.conf`. We recommend a setting of at least 150.
-- Make sure the data nodes are properly listening for network traffic. `postgresql.conf` should have a `listen_addresses` set to include the access node. If this parameter is missing or just `localhost`, change it to `access.example.com`, or the corresponding IP. You can also set it to `"*"` to not restrict where connections can originate from.
-- The superuser role, `postgres` in this example, must have an entry in the `pg_hba.conf` file for the data nodes. This should look something like: ```host    all    postgres    access.example.com    trust``` You can use any CIDR block covering the access node's IP in place of `access.example.com`. For using authentication mechanism's other than `trust`, please see the [authentication guide][data-node-authentication].
+### Update your Postgres configuration
+
+First of all, if you haven't yet run the `timescaledb-tune` utility, it's
+highly recommended that you do so before starting. Do this for all of your
+nodes.
```

Comment:
Suggestion for a less formal version: "It is highly recommended that you run `timescaledb-tune` on all nodes before you start, so if you haven't done this yet, do it now."